temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Using print statements for logging is not recommended. Consider using the 'logging' module to provide better control over logging levels and outputs.
```
import logging

logging.basicConfig(level=logging.INFO)
logging.info("hello")
```